.changeset/small-words-show.md

```diff
@@ -0,0 +1,5 @@
+---
+'@biomejs/biome': minor
+---
+
+added the new rule [`no-unknown-property`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unknown-property.md) from react-rules
```

Comment:
This should also contain examples of valid/invalid code. It also should mention `ignore` getting added to the css rule (maybe in another changeset).

```suggestion
Added the new rule [`noUnknownProperty`](use biome link here) from react-rules
```

Reply:
Done, i need to check for css rule